python/dgl/nn/pytorch/explain/pgexplainer.py

```diff
@@ -30,6 +30,10 @@ class PGExplainer(nn.Module):
           the intermediate node embeddings.
     num_features : int
         Node embedding size used by :attr:`model`.
+    num_hops : int
+        The number of hops for GNN information aggregation.
```

Comment:
"The number of hops for GNN information aggregation." -> "The number of hops for GNN information aggregation, which must match the number of message passing layers employed by the GNN to be explained."